core-common/src/main/java/org/zalando/nakadi/domain/EventOwnerHeader.java

```diff
@@ -70,4 +70,9 @@ public int hashCode() {
         result = 31 * result + value.hashCode();
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "{ name: " + this.name + ", value: " + this.value + " }";
```

Comment:
could log sensitive information

Reply:
Without: `java.lang.AssertionError: expected:<org.zalando.nakadi.domain.EventOwnerHeader@aacf320e> but was:<org.zalando.nakadi.domain.EventOwnerHeader@aacf2a8e>` ;)

Reply:
could you use equals method in the assertions? i would avoid logging that as much as possible

Reply:
OK, let's remove it just in case.